website/docs/docs/deploy/deploy-environments.md

```diff
@@ -39,7 +39,9 @@ In dbt Cloud, each project can have one designated deployment environment, which
 
 ### Semantic Layer
 
-For Semantic Layer-eligible customers, the next section of environment settings is the Semantic Layer configurations. [The Semantic Layer setup guide](/docs/use-dbt-semantic-layer/setup-sl) has the most up-to-date setup instructions!
+For Semantic Layer-eligible customers, the next section of environment settings is the Semantic Layer configurations. [The Semantic Layer setup guide](/docs/use-dbt-semantic-layer/setup-sl) has the most up-to-date setup instructions. 
```

Comment:
Our goal in Q3 is to focus on adoption of the SL, including making it available to developer plan. It might be fudging a bit right now, but can we update the language to just say:

For customers using the Semantic Layer... 

I don't want people to read this and think "oh can I use this yet" the default should just be yes you can if you use dbt.

Reply:
oh that's good to know and very cool!